pkgs/development/php-packages/ssh2/default.nix

```diff
@@ -0,0 +1,19 @@
+{ buildPecl, lib, libssh2 }:
+
+buildPecl rec {
+  version = "1.3.1";
+  pname = "ssh2";
+
+  sha256 = "sha256-kJOh+NJNxlg2AnsOI5xQ3o1err+DlrwzMf3TjF1pr9k=";
+
+  buildInputs = [ libssh2 ];
```

Comment:
I think this should be `nativeBuildInputs`, to be tested.

`libssh2` is only required at build time, not at runtime.

Reply:
That does not sound right. If this is bindings for the library, it must necessarily be used at runtime.

Reply:
Oh, good to know. I though including it was enough to use it in the extension.

Sorry for that if that's the case.

Reply:
It probably only works with `nativeBuildInputs` because we have not enabled `strictDeps` yet.

Nixpkgs manual recently received a [simpler explanation of this topic](https://nixos.org/manual/nixpkgs/unstable/#ssec-stdenv-dependencies-overview) so I recommend to peruse it.

Reply:
Yes, we should revert and use `buildInputs` then.

Reply:
reverted